addons/web/static/src/views/view.js

```diff
@@ -278,6 +278,34 @@ export class View extends Component {
         }
     }
 
+    async loadBanner(bannerRoute) {
+        if (!bannerRoute) {
```

Comment:
I prefer to put this kind of condition outside of the function (at least when the function is called in very few places)

Reply:
(does not make sense to call loadBanner when we know there is no banner to load...)